src/user/related_models/user_model.py

```diff
@@ -35,28 +35,29 @@ def editors(self):
 
 
 class User(AbstractUser):
+    agreed_to_terms = models.BooleanField(default=False)
+    bookmarks = models.ManyToManyField(
+        "paper.Paper", related_name="users_who_bookmarked"
+    )
     country_code = models.CharField(max_length=4, null=True, blank=True)
-    reputation = models.IntegerField(default=100)
-    upload_tutorial_complete = models.BooleanField(default=False)
+    created_date = models.DateTimeField(auto_now_add=True)
     has_seen_first_coin_modal = models.BooleanField(default=False)
     has_seen_orcid_connect_modal = models.BooleanField(default=False)
     has_seen_stripe_modal = models.BooleanField(default=False)
-    agreed_to_terms = models.BooleanField(default=False)
-    created_date = models.DateTimeField(auto_now_add=True)
-    updated_date = models.DateTimeField(auto_now=True)
-    bookmarks = models.ManyToManyField(
-        "paper.Paper", related_name="users_who_bookmarked"
+    invited_by = models.ForeignKey(
+        "self", related_name="invitee", on_delete=models.SET_NULL, null=True, blank=True
     )
-    moderator = models.BooleanField(default=False)
     is_suspended = models.BooleanField(default=False)
+    moderator = models.BooleanField(default=False)
     probable_spammer = models.BooleanField(default=False)
-    suspended_updated_date = models.DateTimeField(null=True)
-    spam_updated_date = models.DateTimeField(null=True)
     referral_code = models.CharField(max_length=36, default=uuid.uuid4, unique=True)
-    invited_by = models.ForeignKey(
-        "self", related_name="invitee", on_delete=models.SET_NULL, null=True, blank=True
-    )
+    reputation = models.IntegerField(default=100)
+    should_display_rsc_balance_home = models.BooleanField(default=True)
```

Comment:
I like your idea of separating out user preferences into another model, but this is fine for now